wai-extra/Network/Wai/Test.hs

```diff
@@ -5,6 +5,7 @@ module Network.Wai.Test
     ( -- * Session
       Session
     , runSession
+    , runSessionWith, ClientState(..), initState
```

Comment:
I'd rather not expose the constructor and fields. Would it work to just export `ClientState`?